app/models/member.rb

```diff
@@ -49,12 +58,20 @@ class Member < ApplicationRecord
     inclusion: { in: SEXES },
     allow_nil: true
 
+  validate :correct_other_income_types
```

Comment:
this is a "custom validator", which is a totally cool thing to do and in many cases necessary. In this case, rails gives us [a validator we can use](http://guides.rubyonrails.org/active_record_validations.html#inclusion):

```
 validates :other_income_types,
    inclusion: { in: OTHER_INCOME_TYPES },
    allow_nil: true

```
that "inclusion" validator will do the same thing that your custom validator is doing.

Reply:
The problem with the Rails validator here is that `other_income_types` is not a normal string but a magic array type.
`t.string "other_income_types", default: [], array: true`

I don't know if you can use the inclusion validator in this case sadly.